src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const list = document.querySelectorAll('li');
+
+for (const el of list) {
+  const span = document.createElement('span');
+
+  el.prepend(span);
+  span.prepend(span.nextSibling);
+}
+
+tree.addEventListener('click', (e) => {
+  const item = e.target.closest('li').children[1];
+
+  item.hidden = !item.hidden;
```

Comment:
small improvement here: it would be great to check if the item was found to avoid error logs in console
```suggestion
  if (item) {
    item.hidden = !item.hidden;
  }
```